### ISSUE TITLE
Using fetching latest results

### ISSUE THREAD

**Issue by tamaszl** [RIPE-NCC/ripe-atlas-cousteau#58]
I'm using fetching latest results described here: https://ripe-atlas-cousteau.readthedocs.io/en/latest/use.html#fetching-latest-results
```
% python3 latest_results.py
{'error': {'detail': 'Method "GET" not allowed.', 'status': 405, 'title': 'Method Not Allowed', 'code': 104}}
```
Method is not allowed. Is this code outdated?

**Comment by chrisamin** [comment]
The docs are incorrect unfortunately, they're missing the msm_id:

```
>>> AtlasLatestRequest(msm_id=1001, probe_ids=[1, 2, 3, 4]).create()
> (True, [{'fw': 4790, 'lts': 326, 'dst_name': '193.0.14.129', 'af': 4, 'dst_addr': '193.0.14.129', 'src_addr': '192.168.178.26', 'proto': 'ICMP', 'ttl': 52, 'size': 20, 'result': [{'rtt': 28.532741}, {'rtt': 27.625734}, {'rtt': 26.895366}], 'dup': 0, 'rcvd': 3, 'sent': 3, 'min': 26.895366, 'max': 28.532741, 'avg': 27.6846136667, 'msm_id': 1001, 'prb_id': 1, 'timestamp': 1645626064, 'msm_name': 'Ping', 'from': '45.138.229.91', 'type': 'ping', 'step': 240, 'st.......
```
You can fix the latest_results.py file by just adding an appropriate "msm_id" value.

I will make sure that the docs are updated and that the next version of cousteau raises an appropriate exception when msm_id isn't specified.

Thanks for your report!

**Comment by chrisamin** [comment]
Fixed in v1.5.0
